test/SlowTests/Issues/RavenDB-15133.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System.Linq;
+using System.Threading.Tasks;
+using FastTests;
+using Raven.Client.Documents.Indexes;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace SlowTests.Issues
+{
+    public class RavenDB_15133 : RavenTestBase
+    {
+        public RavenDB_15133(ITestOutputHelper output) : base(output)
+        {
+        }
+
+        private class Document
+        {
+            public string Id { get; set; }
+            public string Name { get; set; }
+        }
+
+        private class DocumentIndex : AbstractIndexCreationTask<Document>
+        {
+            public class Result
+            {
+                public string Name { get; set; }
+            }
+
+            public DocumentIndex()
+            {
+                Map = doc => from documents in doc
+                             select new Result
+                             {
+                                 Name = documents.Name
+                             };
+            }
+        }
+
+        [Fact]
+        public async Task CanSelectIdField()
```

Comment:
How does this relates to the change you made?

Reply:
forgot to push the test